components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/AppMConstants.java

```diff
@@ -25,6 +25,9 @@
  */
 public final class AppMConstants {
 
+    // Velocity logger name parameter in app-manager.xml
+	public static final String VELOCITY_LOGGER = "VelocityLogger";
```

Comment:
Code is formatted, GitHub doesn't show it properly. Please see the code in the IDE.